ambersim/rl/networks.py

```diff
@@ -0,0 +1,117 @@
+from typing import Sequence
+
+import flax.linen as nn
+import jax
+import jax.numpy as jnp
+from brax.training import distribution, networks, types
+from brax.training.agents.ppo.networks import PPONetworks
+from flax import struct
+
+"""
+Convienience tools for defining neural networks for RL.
+"""
+
+
+class MLP(nn.Module):
```

Comment:
Can we move MLP to a more generic spot that isn't under RL? Maybe under `ambersim/dl/arch.py` or something like that. Completely open to suggestions on path/names.

Reply:
Good idea! I moved it to `ambersim/learning/architectures.py`. Also open to revisions on what names we use exactly